frontend/src/ErrorPopup/index.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import {
+  View, Text, TouchableOpacity, StyleSheet, Image,
+  SafeAreaView, TouchableWithoutFeedback,
+} from 'react-native';
+import PropTypes from 'prop-types';
+
+import Gradient from 'react-native-css-gradient';
+import Popup from '../common/components/Popup';
+import ModalHeader from '../common/components/ModalHeader';
+import CloseButton from './CloseButton';
+
+import errorImage from '../../assets/error.gif';
+import commonColors from '../common/colors';
+
+const styles = StyleSheet.create({
+  background: {
+    flex: 1,
+    backgroundColor: 'rgba(0,0,0,0.6)',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  container: {
+    width: 300,
+    borderWidth: 2,
+    borderColor: 'grey',
+    borderRadius: 6,
+    overflow: 'hidden',
+  },
+  modalHeader: {
+    backgroundColor: 'rgba(255,0,0,0.9)',
+    borderTopStartRadius: 0,
+    borderTopRightRadius: 0,
+    borderWidth: 2,
+    borderColor: 'grey',
+  },
+  mainBody: {
+    padding: 20,
+    width: '100%',
+    alignItems: 'center',
+    backgroundColor: '#000',
+    borderWidth: 2,
+    borderColor: 'grey',
+  },
+  imgStyle: {
+    height: 175,
+    width: 175,
+  },
+  imgContainer: {
+    borderRadius: 10,
+    overflow: 'hidden',
+  },
+  textStyle: {
+    margin: 10,
+    fontSize: 20,
+    color: commonColors.whiteColor,
+    fontWeight: 'bold',
+  },
+});
+
+const gradient = `linear-gradient(-225deg, red 0%, orange 17%, 
```

Comment:
it looks funny with this many colors and with the gif, but I think it would look strange on the final demo since it has nothing to do with the design of our application :/ also the gif is cute but it looks more like a loading gif to me.

let's discuss these things on our next sprint, but I think it just stands out too much from our app's colors and design